src/Tester/Tester.csproj

```diff
@@ -43,62 +43,67 @@
     <AssemblyOriginatorKeyFile>..\Orleans.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bond">
-      <HintPath>$(SolutionDir)packages\Bond.Runtime.CSharp.3.0.7\lib\net45\Bond.dll</HintPath>
```

Comment:
While I agree partially. Only 2 steps and 1 thing was done for this PR. Fighting the tooling is wrong IMHO, specially if that fight comes from some oldish fears of nuget and/or pre nuget. Nuget now works pretty well at engine level and I think that all references should be handled the nuget way and only project references would be the other which should be allowed.

Handcoded $(SolutionDir)packages is evil from contributor perspective.